front/src/modules/ui/input/relation-picker/components/SingleEntitySelectBase.tsx

```diff
@@ -71,9 +76,22 @@ export function SingleEntitySelectBase<
   entitiesInDropdown = entitiesInDropdown.filter((entity) =>
     isNonEmptyString(entity.name.trim()),
   );
+  const theme = useTheme();
 
   return (
     <DropdownMenuItemsContainer ref={containerRef} hasMaxHeight>
+      {noUser && (
+        <DropdownMenuItem onClick={() => onEntitySelected(noUser)}>
```

Comment:
let's actually introduce an optional onEntityUnselected callback as a props of `SingleEntitySelectBase`  a and avoid hacking into the existing onEntitySelected
We will pass it from CompanyPickerCell and UserPicker